src/python/gudhi/hera/wasserstein.cc

```diff
@@ -49,8 +147,9 @@ PYBIND11_MODULE(wasserstein, m) {
             order (float): Wasserstein exponent W_q
             internal_p (float): Internal Minkowski norm L^p in R^2
             delta (float): Relative error 1+delta
+            matching (bool): if ``True``, computes and returns the optimal matching between X and Y, encoded as a (n x 2) np.array [...[i,j]...], meaning the i-th point in X is matched to the j-th point in Y, with the convention that (-1) represents the diagonal.
 
         Returns:
-            float: Approximate Wasserstein distance W_q(X,Y)
+            float|Tuple[float,numpy.array]: Approximate Wasserstein distance W_q(X,Y), and optionally the corresponding matching
```

Comment:
On [POT wasserstein version](https://gudhi.inria.fr/python/latest/wasserstein_distance_user.html#gudhi.wasserstein.wasserstein_distance), we also specify that it returns `None` when cost is +inf. Do you think it worth it to specify it here ?

Reply:
done